addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1691,7 +1694,8 @@ const Wysiwyg = Widget.extend({
         this.toolbar.$el.find('#create-link').toggleClass('d-none', !range || spansBlocks);
         // Only show the media tools in the toolbar if the current selected
         // snippet is a media.
-        const isInMedia = $target.is(mediaSelector);
+        const isInMedia = $target.is(mediaSelector) && e.target &&
+            (e.target.isContentEditable || e.target.parentElement && e.target.parentElement.isContentEditable);
```

Comment:
Are we sure that `$target` is equivalent to `e.target` ?

Reply:
A few lines above:
```js
const $target = e ? editorWindow.$(e.target) : editorWindow.$();
```
So either it's not defined (hence the `e.target` check) or it's equivalent.